.github/workflows/main.yml

```diff
@@ -47,12 +47,24 @@ jobs:
     with:
       logs-artifact: driver-build-failures
 
-  build-collector-slim:
+  build-collector-slim-amd64:
     uses: ./.github/workflows/collector-slim.yml
     needs: init
     with:
       collector-tag: ${{ needs.init.outputs.collector-tag }}
       collector-image: ${{ needs.init.outputs.collector-image }}
+      arch: amd64
+    secrets: inherit
+
+  build-collector-slim-ppc64le:
+    uses: ./.github/workflows/collector-slim.yml
+    needs:
+      - init
+      - build-collector-slim-amd64
```

Comment:
Why is slim-ppc64le depending on slim-amd64 ?

Reply:
It was to force them to run sequentially rather than in parallel, mostly as a way of ensuring that amd64 runs first. There is a possibility that there aren't enough workers to run both at the same time, which could mean that ppc64le runs first, which is back to being extremely slow again (I did see this in testing.) 

It's probably not strictly necessary though

Reply:
Understood, though I never saw such a case happen in GHA.

I believe that we could leave those run in parallel and keep the graph clean.